packages/mobile/src/navigator/Navigator.tsx

```diff
@@ -296,6 +296,7 @@ const AppStack = createStackNavigator(
     [Screens.PhotosEducation]: { screen: PhotosEducation },
     [Screens.GoldEducation]: { screen: GoldEducation },
     [Screens.FeeEducation]: { screen: FeeEducation },
+    [Screens.FeeExchangeEducation]: { screen: FeeExchangeEducation },
```

Comment:
Added so we navigate back to exchange confirmation card, rather than always going to exchange. 
Ex flow this fixes: Click exchange notification from home screen, then click FeeExchangeEducation, expect to return back to exchange confirmation card but instead go to the exchange

Reply:
Maybe leave a comment explaining this?